src/Microsoft.Bcl.HashCode/src/Interop.GetRandomBytes.cs

```diff
@@ -3,12 +3,21 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
+using System.Runtime.InteropServices;
+using System.Security.Cryptography;
 
 internal partial class Interop
 {
     internal static unsafe void GetRandomBytes(byte* buffer, int length)
     {
-        byte[] bytes = Guid.NewGuid().ToByteArray();
-        buffer = (byte*)BitConverter.ToUInt32(bytes, 0);
+        if (!LocalAppContextSwitches.UseNonRandomizedHashSeed)
```

Comment:
It's a little strange to have this here rather than at the call site.  Is this just because we're trying to keep the call site unchanged?  If so, ok.

Reply:
That's right, the first iteration had the change at the callsite but the problem is that it is a shared file so we had to add a bunch of ifdefs in there. Doing it this way makes the fix "cleaner" as it is targeted to only one file which only gets compiled for this package.